adapters/freewheelssp/freewheelssp.go

```diff
@@ -12,11 +12,11 @@ import (
 	"github.com/prebid/prebid-server/v2/openrtb_ext"
 )
 
-type adapter struct {
-	endpoint string
+type Adapter struct {
```

Comment:
This does not have to be an exported field. Please change it to use lowercase `type adapter struct`